api/user/index.ts

```diff
@@ -4,6 +4,9 @@ import { getConfigValue } from '../configuration'
 import { CASE_SHARE_PERMISSIONS, SESSION_TIMEOUTS } from '../configuration/references'
 
 export async function getUserDetails(req, res: Response, next: NextFunction) {
+  if (!req.session || !req.session.passport || !req.session.passport.user) {
+    res.send({});
```

Comment:
_Question:_
Should you `return` here? If you don't do so, the code below (line 11 onwards) will execute and, given you already know that there is _isn't_ a `req.session.passport.user`, you'll end up in the `catch` block... which probably isn't what you want.
```suggestion
    return res.send({});
```

Reply:
yes Paul you are right (as always), I should return the object. 
Done the change
thanks

Reply:
Definitely not always! 😂 

I see you've changed it... and added `.status(200)`. Shouldn't it be `.status(401)` since the user isn't authenticated?

Reply:
regarding 401 code instead of 200; if we return 401 then we get error in console as unauthorized (401) which we don't want as we are redirecting to to Idam login if we don't find user details.